playground/nextjs-pages/src/pages/_app.tsx

```diff
@@ -23,12 +23,11 @@ const NAVIGATION: Navigation = [
     title: 'Main items',
   },
   {
-    segment: '/',
```

Comment:
This one can be omitted in this case, not sure if we want that.

Reply:
Might be good to omit for ease of understandability